packages/oui/src/ScrollProgress/ScrollProgress.js

```diff
@@ -12,21 +12,53 @@ export function calculateVerticalProgress(bounds, topOffset = 0, bottomOffset =
 }
 
 const ScrollProgress = React.forwardRef(function ScrollProgress(props, ref) {
-  const { onChange, onEnter, onExit, ...other } = props
+  const { friction = 0, onChange, onEnter, onExit, precision = 0.001, ...other } = props
 
   const rootRef = React.useRef(null)
 
+  const stateRef = React.useRef({
+    currentInnerProgress: 0,
+    currentProgress: 0,
+    isEasing: false,
+    targetInnerProgress: 0,
+    targetProgress: 0,
+  })
+
+  const handleRaf = React.useCallback(() => {
```

Comment:
I would call this function one of the following: 

```suggestion
  const handleAnimationFrame = React.useCallback(() => {
```
```suggestion
  const handleAf = React.useCallback(() => {
```

I think `Raf` stands for `requestAnimationFrame`, but i think the `requestAnimationFrame` is a function requesting the actual animation frame.

I think that the name `handleRaf` sounds like a wrapper for the actual `requestAnimationFrame` function and no a callback for it.

Reply:
Sure thing, could go for `handleAnimationFrame` :)